lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/config.cpp

```diff
@@ -27,13 +27,177 @@
 #include "json_helpers.h"
 #include "logging.h"
 
+using namespace pew::eos::config;
+using namespace pew::eos::json_helpers;
+
+ /**
+  * @brief Reads an EOS configuration file as a JSON value.
+  *
+  * Retrieves the specified configuration file, loads its contents, and parses it into a JSON structure.
+  *
+  * @param config_filename The name of the configuration file.
+  * @return A pointer to a `json_value_s` representing the configuration data, or `nullptr` if parsing fails.
+  */
+json_value_s* read_eos_config_as_json_value_from_file(std::string config_filename);
+
+EOSConfig eos_config_from_json_value(json_value_s* config_json)
+{
+    // Create platform instance
+    json_object_s* config_json_object = json_value_as_object(config_json);
+    json_object_element_s* iter = config_json_object->start;
+    EOSConfig eos_config;
+
+    while (iter != nullptr)
+    {
+        if (!strcmp("productName", iter->name->string))
```

Comment:
This surprises me a bit. "If productName cannot be compared to the current iterator, then do things." Is that what "!strcmp" is implying here? I would have expected it to do the thing it's in if it were *true*.

Reply:
Yeah, me too - but if `strcmp` returns `0` if the two strings are equal, and in C++ `0` equates to `false`. I _also_ don't like how it looks 😆